<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>insertWindow</class>
 <widget class="QMainWindow" name="insertWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>391</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>Single Insertion</string>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>361</width>
       <height>241</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>             Book number:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="editBno"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>                Category:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="editCategory"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>                   Title:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="editTitle"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>                  Press:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>                   Year:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="editYear"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>                 Author:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="editAuthor"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>                  Price:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="editPrice"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>                  Total:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="editTotal"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>                 Stock:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="editStock"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="editPress"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="insertBtn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>340</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Insert</string>
     </property>
    </widget>
    <widget class="QLabel" name="stateLbl">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>340</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>State:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>301</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string> Multiple Insertion</string>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Select File :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editFileName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="mulStateLbl">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>170</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>State: </string>
     </property>
    </widget>
    <widget class="QToolButton" name="selectFileBtn">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>70</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="mulInsertBtn">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>170</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>797</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile_F">
    <property name="title">
     <string>File(F)</string>
    </property>
   </widget>
   <addaction name="menuFile_F"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
